recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "cp2k" %}
-{% set version = "6.1.0" %}
+{% set version = "7.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
The default tar-ball doesn't work because it doesn't contain the needed submodules. Please use https://github.com/cp2k/cp2k/releases/download/v7.1.0/cp2k-7.1.tar.bz2 instead.